### PR TITLE
6.4.x Bug fixes

### PR DESCRIPTION
**Pull request by bbrodt**
Fixes the following BZs
	https://bugzilla.redhat.com/show_bug.cgi?id=1261874
	https://bugzilla.redhat.com/show_bug.cgi?id=1269106

### CODE REVIEW

**Comment by mswiderski** [comment]
@krisv do you mind reviewing it and +1 so it can get merged?

@bbrodt shall this go into 6.4.x only or was it already applied on master?

**Comment by bbrodt** [comment]
Requested by Paul Leacu for JBDSIS stack.

**Comment by bbrodt** [comment]
These are for 6.4.x - have not cherry picked into master yet.
I created a "New Pull Request" for each of these commits, but for some reason they are all included in the same PR - I don't get it :(

**Comment by krisv** [comment]
+1

Bob, just one remark, it might be best to wait before you put a BZ to modified until the PR is merged, and include the commit id there as well.

File: drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java [hunk 1]
```diff
@@ -151,8 +152,7 @@ protected String httpDelete(String request) throws IOException {
 	protected String httpPost(String request, JsonObject body) throws IOException, RuntimeException {
 		String host = getKieRESTUrl();
 		URL url = new URL(host + "/" + request);
-		System.out.println("[POST] "+url.toString());
-		System.out.println("[POST] body: "+body);
+		System.out.println("[POST] "+url.toString()+" body: "+body);
```
Comment (krisv) [inline drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java]: Are System.out statements still actually useful?  Shouldn't they be removed or replaces by some more configurable logging?

Comment (bbrodt) [inline drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java]: You're right, the sysouts should be removed - they were useful for
debugging but need to go now. I'll create another commit for this.

On Thu, Feb 18, 2016 at 1:12 PM, Kris <notifications@github.com> wrote:

> In
> drools-eclipse/org.kie.eclipse/src/main/java/org/kie/eclipse/server/KieServiceDelegate.java
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/42#discussion_r53375411>
> :
>
> > @@ -151,8 +152,7 @@ protected String httpDelete(String request) throws IOException {
> >  	protected String httpPost(String request, JsonObject body) throws IOException, RuntimeException {
> >  		String host = getKieRESTUrl();
> >  		URL url = new URL(host + "/" + request);
> > -		System.out.println("[POST] "+url.toString());
> > -		System.out.println("[POST] body: "+body);
> > +		System.out.println("[POST] "+url.toString()+" body: "+body);
>
> Are System.out statements still actually useful? Shouldn't they be removed
> or replaces by some more configurable logging?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/droolsjbpm/droolsjbpm-tools/pull/42/files#r53375411>.
>



-- 
________________________
Robert ("Bob") Brodt
Senior Software Engineer
JBoss by Red Hat

**Comment by bbrodt** [comment]
Ah, I didn't realize I needed a new topic branch for each commit/bug, otherwise the commits just pile up in the same PR...next time I know ;)

**Comment by mswiderski** [comment]
merged into 6.4.x, thanks @bbrodt

**Pull request closed without merging**